package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "math-magicians",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
```

Comment:
- [ ] Kindly fix the error which is displayed when the app starts running.

https://user-images.githubusercontent.com/70269507/198075362-f6ed5fd2-4d72-448c-a1b5-74bff0784b54.mov

Reply:
Okay @sja-thedude I will do that now. Thank you